### PR TITLE
Add Ledger docs

### PR DESCRIPTION
**Pull request by gabrielKerekes**

### CODE REVIEW

File: doc/ledger-hw-wallet.md [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+- Verify Ledger address
+- Sign Ledger transactions
+
+Unsupported features:
+- message signing
+- NTP1 tokens
+- cold-staking
+
+## Add Ledger address
+Available via the "New address" button in the "Receive" tab in Qt Wallet by checking the "Ledger address" checkbox in the "New Address" dialog. The user chooses the account index and address index which are used to build the BIP-32 address derivation path (e.g. "m/44'/146'/0'/0/0"). Along with the the payment address (m/44'/146'/0'/**0**/0) a change address (m/44'/146'/0'/**1**/0) is imported as well. Ledger address labels **must** be unique i.e. a Ledger account label cannot be used by **any** other wallet account.
```
Comment (davidmisiak) [inline doc/ledger-hw-wallet.md]: > Ledger address labels **must** be unique i.e. a Ledger account label cannot be used by **any** other wallet account.

Nit: It also needs to be non-empty.

File: doc/ledger-hw-wallet.md [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+
+Also available via RPC - "addledgeraddress <accountindex> <addressindex> <label>".
```
Comment (davidmisiak) [inline doc/ledger-hw-wallet.md]: This seems to be broken in markdown? (also below)
image

File: doc/ledger-hw-wallet.md (hunk 1 above)
Comment (davidmisiak) [reply doc/ledger-hw-wallet.md]: Typo: `with the the payment address`

File: doc/ledger-hw-wallet.md [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+
+*Note: when adding a Ledger address which might have already been used a blockchain rescan is required to discover the address's funds.*
+
+## Verify Ledger address
+Available via the "Verify Address" button in the "Receive" tab in Qt Wallet. Displays the Ledger address on the screen and then also on Ledger.
+
+Also available via RPC - "verifyledgeraddress <accountindex> <ischange> <addressindex> <expectedaddress>".
+
+## Sign Ledger transactions
+Available by checking the "Pay from a Ledger address" in the "Send" tab in Qt Wallet. A Ledger transaction can only contain inputs from a single **Ledger** payment address and its corresponding change address. This is why a "Pay From" Ledger account has to be selected for Ledger transactions. This also means that funds from the "internal" wallet accounts cannot be sent with along with Ledger funds in the same transactions.
```
Comment (davidmisiak) [inline doc/ledger-hw-wallet.md]: Typo: `with along with`

**Review by davidmisiak — COMMENTED** [review]
A few more things that we could explicitly mention:
- NTP1 tokens should not be sent to a Ledger address because they will become unspendable.
- Importing an existing Ledger account requires a manually triggered rescan.

File: doc/ledger-hw-wallet.md (hunk 1 above)
Comment (gabrielKerekes) [reply doc/ledger-hw-wallet.md]: https://github.com/vacuumlabs/neblio/pull/2/commits/cb9d22b075d08118327b77301b2e183d58aebfb7

File: doc/ledger-hw-wallet.md (hunk 2 above)
Comment (gabrielKerekes) [reply doc/ledger-hw-wallet.md]: Nice catch, thanks. https://github.com/vacuumlabs/neblio/pull/2/commits/cb9d22b075d08118327b77301b2e183d58aebfb7

File: doc/ledger-hw-wallet.md (hunk 3 above)
Comment (gabrielKerekes) [reply doc/ledger-hw-wallet.md]: https://github.com/vacuumlabs/neblio/pull/2/commits/cb9d22b075d08118327b77301b2e183d58aebfb7

**Comment by gabrielKerekes** [comment]
> NTP1 tokens should not be sent to a Ledger address because they will become unspendable.

Do you think this is enough? https://github.com/vacuumlabs/neblio/pull/2/commits/a8df9d70796eb32f2395338540db1508a30f74e3

> Importing an existing Ledger account requires a manually triggered rescan.

Already mentioned [here](https://github.com/vacuumlabs/neblio/pull/2/files#diff-70022e8f4585991f34c847a7eb3f63047108e07896a6d6734a28b0b81e8dc07eR18).

**Review by davidmisiak — APPROVED** [review]

**Pull request merged**